### PR TITLE
Label modal

### PR DESCRIPTION
**Pull request by jamesw6688**
deleted "upload data" option for text box; finalized the style modal for text box

### CODE REVIEW

File: src/api/api.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import axios from 'axios';
 
 const client = axios.create({
-	baseURL: "https://peagle-backend.herokuapp.com"
-	// baseURL: "http://localhost:5000"
+	//baseURL: "https://peagle-backend.herokuapp.com"
```
Comment (nfried16) [inline src/api/api.js]: use this link

File: src/components/dashboards/Grid.jsx [hunk 2]
```diff
@@ -39,7 +39,7 @@ const Grid = props => {
                         h: el.h
                     }}
                     style={{
-                        padding: "1rem",
+                        // padding: "1rem",
```
Comment (nfried16) [inline src/components/dashboards/Grid.jsx]: this is what caused the issue. Why did you do this??

File: src/components/dashboards/Chart.jsx [hunk 3]
```diff
@@ -13,30 +14,42 @@ const Chart = props => {
...
                     style={{
                         position: "absolute",
                         right: "2px",
                         top: 0,
-                        cursor: "pointer"
+                        cursor: "pointer",
                     }}
                     onClick={() => props.onRemoveItem(props.el)}
                 >
-                    x
```
Comment (nfried16) [inline src/components/dashboards/Chart.jsx:39]: you deleted the delete button, we need this!

File: src/components/dashboards/Dashboards.css [hunk 4]
```diff
@@ -51,4 +51,15 @@
...
+.icon:hover {
+  background-color: #D3D3D3;
+}
+
+.icon {
+  font-size: 215%;
+  margin: 5px;
+  padding: 5px;
+  border-radius: 3px;
 }
```
Comment (nfried16) [inline src/components/dashboards/Dashboards.css:65]: I don't think these are ever used, so delete them

File: src/components/widgetSelection/WidgetDataEntry.jsx [hunk 5]
```diff
@@ -139,6 +138,8 @@ class WidgetDataEntry extends React.PureComponent {
 
 
 		const dataProps = content ? { data: content, ...this.state.axes } : {};
+		console.log("check!")
```
Comment (nfried16) [inline src/components/widgetSelection/WidgetDataEntry.jsx]: delete these logs

File: src/components/dashboards/DraftWidgetModal.jsx [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+import './Dashboards.css';
+import { FontSizeOutlined, AlignCenterOutlined, AlignLeftOutlined, AlignRightOutlined, BoldOutlined } from '@ant-design/icons';
+
+const DraftWidgetModal = props => {
+
+    const [font, setFont] = useState(props.el.font);
+    const [align, setAlign] = useState(props.el.align);
+    const [bold, setBold] = useState(props.el.bold);
+
+    console.log(font, align, bold)
```
Comment (nfried16) [inline src/components/dashboards/DraftWidgetModal.jsx]: delete this

**Review by nfried16 — CHANGES_REQUESTED** [review]
make changes in comments

File: src/components/dashboards/Dashboards.css (hunk 4 above)
Comment (jamesw6688) [reply src/components/dashboards/Dashboards.css:65]: They are for the style of the text box modal. I think we need them

**Pull request closed without merging**